src/public-assets/js/dependent-dropdown.js

```diff
@@ -0,0 +1,21 @@
+var Dropdown = (function ($) {
+
+    return {
+        init: function (formEl) {
+            $(formEl).on('change', 'select.dep-drop',  function () {
```

Comment:
@kenny-wealth, a couple of comments here: 
- I believe you can just attach the event listener to `document` or `window`. That way there would there's no need to call this for every form. 
- Also, I think the use of the class name `dep-drop` for each drop-down can be better served with a data- attribute, especially since that class name doesn't carry any specific style. A possible example would be `data-dependent-dropdown`.

Reply:
Fixed